### PR TITLE
Fix videoTileSizeDidChange was called when frame is null

### PR DESCRIPTION
**Pull request by hokyungh**
### Description of changes:
When we call `pauseRemoteVideoTile(tileId)`, it notifies builder the `videoTileSizeDidChange` with 0x0. This should fix that issue. Verify that I am no longer receiving callback when frame is null.

### Testing done:
Pause the video and verify that I am no longer seeing the 0x0 logs.

#### Manual test cases (add more as needed):

- [X] Join meeting without CallKit
- [] Join meeting with CallKit as Incoming
- [ ] Join meeting with CallKit as Outgoing
- [X] Leave meeting
- [X] Rejoin meeting
- [ ] See metrics
- [ ] See attendee presence
- [X] Send audio
- [X] Receive audio
- [X] Mute self
- [X] Unmute self
- [ ] See local mute indicator when muted
- [ ] See remote mute indicator when other is muted
- [ ] Enable and disable local video
- [X] Enable and disable remote video from remote side
- [ ] Send local video
- [X] Pause and resume remote video
- [ ] Switch local camera
- [ ] Receive remote screen share

#### Integration validation (required before release)

- [ ] Unit tests pass
- [ ] Build SDK against simulator with release options
- [ ] Build SDK against device with release options
- [ ] Build SDK against simulator with debug options
- [ ] Build SDK against device with debug options
- [ ] Test Demo against simulator with SDK (debug build) in workspace
- [ ] Test Demo against device with SDK (debug build) in workspace
- [ ] Test DemoObjC against simulator with SDK (debug build) in workspace
- [ ] Test DemoObjC against device with SDK (debug build) in workspace
- [ ] Test Demo against simulator with SDK.framework (release build)
- [ ] Test Demo against device with SDK.framework (release build)
- [ ] Test DemoObjC against simulator with SDK.framework (release build)
- [ ] Test DemoObjC against device with SDK.framework (release build)

### Screenshots, if available:

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+## [Unreleased]
+
+### Fixed
+* Fixed a bug where `videoTileSizeDidChange` is called even when frame is null
```
Comment (tianyux-amz) [inline CHANGELOG.md:4]: Same comment for the iOS change, should this be a breaking change?

Also, probably rephrase this as something like "Fixed a bug that `videoTileSizeDidChange()` being called with width=0 and size=0 when the video was paused", as builders may not have insights on what does "frame is null" means. Same for Android changelog.

Comment (hokyungh) [reply CHANGELOG.md]: updated

**Review by zhinang-amazon — APPROVED** [review]

**Review by alnlau — APPROVED** [review]
